### ISSUE TITLE
Login Screen: UI Glitches 

### ISSUE THREAD

**Issue by ptyagicodecamp** [projectwife/mtesitoo-android#123]
1. Login screen is a bit glitchy. Sometimes keyboard doesn't open when username textbox is focused. User has to click "Next", and then again click on "Username" textbox to access keyboard.

2. Login screen seems to freeze when server authentication is taking place in-background. Its important give user indication that it's in progress and not frozen. A progress bar/hour glass will be helpful.

3. [Aesthetic] "LogIn", "New User" and "Forgot Password" button looks ancient. Will be nice to smooth them out.

**Comment by BassMunkee** [comment]
1. I didn't face this issue in the 5-6 times I tried to login.
2. Noticed this issue once when my network was slow

Another issue I noticed on my device was that the Login button was almost blocked by the keyboard (screenshot attached).
The device is Samsung S6 (2560x1440) running Nougat and Google Keyboard.

![screenshot_20170728-234734](https://user-images.githubusercontent.com/16437515/28730964-ac5ee89c-73ef-11e7-8985-f52b45f0bacc.png)
